support/rbac.go

```diff
@@ -202,3 +202,33 @@ func CreateUserClusterRoleBinding(t Test, userName string, roleName string) *rba
 
 	return rb
 }
+
+func CreateUserNamespacedRoleBinding(t Test, userName string, namespace *corev1.Namespace, roleName string) *rbacv1.RoleBinding {
```

Comment:
Thinking how to distinguish this function and `CreateUserRoleBinding`.
We should indicate that in this case we bind cluster role.

Reply:
Yes, updated function name to `CreateUserRoleBindingWithClusterRole` !